File: bot.php
<?php
// التوكن الخاص بالبوت
$botToken = "7913260818:AAGT5rKbYr6RdKrO5O3OKmBKbSpsdSZ_Z5Y";

// الحصول على البيانات القادمة من Telegram
$update = file_get_contents("php://input");
$updateArray = json_decode($update, true);

// استخراج المعلومات الأساسية
$chatId = $updateArray['message']['chat']['id'];
$text = $updateArray['message']['text'];

// التحقق إذا كان المستخدم ضغط على /start
if ($text === "/start") {
    $message = "مرحبًا! اضغط على الرابط التالي للوصول إلى صفحة جمع البيانات:\n\n";
    $message .= "<a href='https://hanoybot.github.io/Data-collection/'>اضغط هنا</a>";

    // إرسال الرد إلى المستخدم
    file_get_contents("https://api.telegram.org/bot$botToken/sendMessage?chat_id=$chatId&text=" . urlencode($message) . "&parse_mode=HTML");
}
?>

File: bot.py
from telegram import Update
from telegram.ext import Application, CommandHandler, ContextTypes

# بيانات البوت
BOT_TOKEN = "7782026361:AAHmzG4VaYRFrVMxdNLH8K7nGkLGPwIr7pY"

# دالة الاستجابة للأمر "/start"
async def start_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
    chat_id = update.effective_chat.id
    message = "مرحبًا! هذا هو الرابط الخاص بك:\n\nhttps://hanoybot.github.io/Data-collection/"
    await context.bot.send_message(chat_id=chat_id, text=message)

def main():
    # إنشاء تطبيق البوت
    application = Application.builder().token(BOT_TOKEN).build()

    # إضافة أمر "/start"
    application.add_handler(CommandHandler("start", start_command))

    # بدء تشغيل البوت
    print("البوت يعمل الآن!")
    application.run_polling()

if __name__ == "__main__":
    main()
